bin/s2nd.c

```diff
@@ -394,11 +394,7 @@ int handle_connection(int fd, struct s2n_config *config, struct conn_settings se
     /* TODO: However, we should expect to receive a close_notify from the peer and shutdown gracefully. */
     /* Please see tracking issue for more detail: https://github.com/aws/s2n-tls/issues/2692 */
     s2n_blocked_status blocked;
-    int shutdown_rc = s2n_shutdown(conn, &blocked);
-    if (shutdown_rc == -1 && blocked != S2N_BLOCKED_ON_READ) {
-        fprintf(stderr, "Unexpected error during shutdown: '%s'\n", s2n_strerror(s2n_errno, "NULL"));
-        exit(1);
-    }
+    s2n_shutdown(conn, &blocked);
```

Comment:
Could you create an issue to track for these changes and add  it in the comments here?

Reply:
She did create one: https://github.com/aws/s2n-tls/issues/2846

But I thought you'd worked around that issue. What ultimately led you to roll back the shutdown error handling?

Reply:
Shutdown errors because there is one last message to read (the NST). When I got the client to read that last message by calling echo on the last iteration of the reconnect loop, shutdown does not error. However, this ended up causing an integ v1 test to hang, because they were testing TLS1.3 and session tickets are turned on by default for s2nc. Essentially, I can't read that last NST message without affecting older tests.